docs/src/apalache/known-issues.md

```diff
@@ -0,0 +1,16 @@
+# Known issues
+
+This page collects known issues that were reported by the users.
+
+## Deadlock detection
+
+Deadlock detection is imprecise. It may report false negatives, see [Issue
+711][].
+
+**Affected versions:** 0.11.x, 0.15.x
```

Comment:
Does this mean it the issue was introduced in 0.11? Or do we think it's always been present?

Reply:
I think it was always present, as it has never been our focus.

Reply:
That's what I figured! Thanks :)